devel/Dockerfile

```diff
@@ -57,6 +58,23 @@ RUN if [ "$TARGETPLATFORM" != "linux/arm/v7" ]; then \
         cd ../.. && rm -rf qemu; \
     fi
 
-ENV PATH="/opt/phoenix/utils:${PATH}"
+# install Jlink tools for 32-bit armv7 architecture
+RUN if [ "$TARGETPLATFORM" = "linux/arm/v7" ]; then \
+        mkdir -p /opt/SEGGER && \
+        (cd /opt/SEGGER && \
+        wget --post-data "accept_license_agreement=accepted" https://www.segger.com/downloads/jlink/JLink_Linux_arm.tgz && \
+        tar xf JLink_Linux_arm.tgz && \
+        chmod a-w JLink_Linux_V* && \
+        echo "$(ls -d JLink_Linux_V*)" > version && \
+        mv JLink_Linux_V* JLink_Linux_arm && \
+        mkdir -p /etc/udev/rules.d && \
+        cp JLink_Linux_arm/99-jlink.rules /etc/udev/rules.d/99-jlink.rules); \
```

Comment:
can we use USB devices freely from docker environment or we need to start docker image with special options?

Reply:
Yes, we can use USB devices freely without special options (I've tested it by removing jlink rules from rpi and runner triggered in docker container worked fine)

Reply:
ok, nice.

Reply:
Oh, sorry I've tested it on rpi, where docker image starts with `--privileged` arg. It's needed to use JLink tools (verified by `JLinkGDBServer -device nrf9160 -if swd -port 2331`).